quantum/quantum.c

```diff
@@ -784,6 +792,26 @@ bool process_record_quantum(keyrecord_t *record) {
       return false;
     }
 
+    case KC_LCPO: {
```

Comment:
You can move this stuff above the `SFTENT` case to be closer to the original Space Cadet keycodes.